src/libs/actions/Policy/Category.ts

```diff
@@ -542,8 +542,11 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
     const policyCategoryExpenseRule = CategoryUtils.getCategoryExpenseRule(policy?.rules?.expenseRules ?? [], policyCategory.oldName);
     const approvalRules = policy?.rules?.approvalRules ?? [];
     const expenseRules = policy?.rules?.expenseRules ?? [];
+    const mccGroup = policy?.mccGroup ?? [];
     const updatedApprovalRules: ApprovalRule[] = lodashCloneDeep(approvalRules);
     const updatedExpenseRules: ExpenseRule[] = lodashCloneDeep(expenseRules);
+    const clonedMccGroup: never[] | Record<string, MccGroup> = lodashCloneDeep(mccGroup);
+    const updatedMccGroups = CategoryUtils.updateCategoryInMccGroups(clonedMccGroup, policyCategory.oldName, policyCategory.newName);
```

Comment:
```suggestion
    const updatedMccGroups = CategoryUtils.updateCategoryInMccGroups(clonedMccGroup, policyCategory.oldName, policyCategory.newName);
```

Correct me if I'm wrong, but shouldn't this be called `updatedMccGroup` and `updateCategoryInMccGroup` since we're updating only one mccGroup?